WorkflowUIReactiveSwift/Sources/WorkflowHostingController+OutputSignal.swift

```diff
@@ -0,0 +1,29 @@
+#if canImport(UIKit)
+import Combine
+import ReactiveSwift
+import WorkflowUI
+
+extension WorkflowHostingController {
+    public var output: Signal<Output, Never> {
```

Comment:
I wonder if we can avoid spinning up a whole extra target for less than 30 lines. What if both `WorkflowHost` and `WorkflowHostingController` conformed to a protocol with `outputPublisher`, and we added `output` in an extension to that protocol in `WorkflowReactiveSwift`?

Reply:
Good idea.  I'll give that a shot.

Reply:
Done